### PR TITLE
Adding support for 'ON x GOTO a,b,c,d'.

### PR DESCRIPTION
**Pull request by pauwell**
I implemented the changes to my previous solution and added a test.
If this is fine with you so far I would like to add <code>ON...GOSUB</code> afterwards.

### CODE REVIEW

File: test/on-goto-test.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ON 5 GOTO a,b
+PRINT "done"
+END
+a:
+  END
+b:
+  END
+`,`
+done
+`);
```
Comment (flagxor) [inline test/on-goto-test.js]: Newline?

File: wwwbasic.js [hunk 2]
```diff
@@ -1898,20 +1898,24 @@
...
+        curop += 'ip = labels[[';
+        while (!(EndOfStatement())) { 
+          curop += '\'' + String(tok) + '\'';
           Next();
-          // TODO: Implement.
-        } else {
-          Throw('Expected goto');
+          if(EndOfStatement()) {
+            curop += '][(('+name+')|0) - 1]] || ip;\n';
+          }else {
```
Comment (flagxor) [inline wwwbasic.js]: This will likely fail the linter.

**Comment by pauwell** [comment]
There seems to be a problem with the line endings when comparing the Copyright headers. I will convert my dos line endings to unix.

**Comment by flagxor** [comment]
Ah, interesting with the line endings.
I'll take a look at making it more flexible at dealing with autocrlf.

Thanks for implementing on-goto!

lgtm

**Pull request merged**